ql/src/java/org/apache/hadoop/hive/ql/ddl/database/create/CreateDatabaseAnalyzer.java

```diff
@@ -70,19 +73,43 @@ public void analyzeInternal(ASTNode root) throws SemanticException {
         managedLocationUri = unescapeSQLString(childNode.getChild(0).getText());
         outputs.add(toWriteEntity(managedLocationUri));
         break;
+      case HiveParser.TOK_DATACONNECTOR:
+        type = "REMOTE";
+        // locationUri = "REMOTE_DATABASE"; // TODO
```

Comment:
this is a dead comment. I had this deleted in the local set of changes.
Essentially, the location is a required field on the Database. But this is handled in the CreateDatabaseOperation.makeQualifiedPath() where the path of the DB is set to what it would be for a NATIVE DB. There is no harm in using that path.